### PR TITLE
Misc fixes for CCIP1.6 staging

### PR DESCRIPTION
**Pull request by connorwstein**
- Add BalanceAt for gas funding
- Add cap reg view
- Remove transaction receipt retries (ethereum.NotFound expected), let Confirm handle that specially
- Fix JD connectivity

### CODE REVIEW

File: integration-tests/deployment/ccip/view/chain.go [hunk 1]
```diff
@@ -16,11 +16,12 @@ type ChainView struct {
...
-	NonceManager map[string]v1_6.NonceManagerView `json:"nonceManager,omitempty"`
-	RMN          map[string]v1_6.RMNRemoteView    `json:"rmn,omitempty"`
-	OnRamp       map[string]v1_6.OnRampView       `json:"onRamp,omitempty"`
-	OffRamp      map[string]v1_6.OffRampView      `json:"offRamp,omitempty"`
+	FeeQuoter          map[string]v1_6.FeeQuoterView    `json:"feeQuoter,omitempty"`
+	NonceManager       map[string]v1_6.NonceManagerView `json:"nonceManager,omitempty"`
+	RMN                map[string]v1_6.RMNRemoteView    `json:"rmn,omitempty"`
+	OnRamp             map[string]v1_6.OnRampView       `json:"onRamp,omitempty"`
+	OffRamp            map[string]v1_6.OffRampView      `json:"offRamp,omitempty"`
+	CapabilityRegistry map[string]v1_6.CapRegView       `json:"capabilityRegistry,omitempty"`
```
Comment (AnieeG) [inline integration-tests/deployment/ccip/view/chain.go:24]: The contract is versioned 1.0 though

File: integration-tests/deployment/ccip/view/v1_6/capreg.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+	var capViews []CapabilityView
+	for _, capability := range caps {
+		capViews = append(capViews, CapabilityView{
+			LabelledName:   capability.LabelledName,
+			Version:        capability.Version,
+			ConfigContract: capability.ConfigurationContract,
+		})
+	}
+	return CapRegView{
+		ContractMetaData: tv,
```
Comment (AnieeG) [inline integration-tests/deployment/ccip/view/v1_6/capreg.go:42]: capviews?

File: integration-tests/deployment/ccip/view/chain.go (hunk 1 above)
Comment (connorwstein) [reply integration-tests/deployment/ccip/view/chain.go:24]: yeah this is a bit of a weird case, its shared/multi-product 1.0 contract which we introduced in 1.6. I think still makes most sense to include in 1.6

Comment (AnieeG) [reply integration-tests/deployment/ccip/view/chain.go:24]: How about adding a comment? so that we don't end up changing it later

**Review by AnieeG — DISMISSED** [review]

**Review by AnieeG — APPROVED** [review]

**Review by ogtownsend — APPROVED** [review]

**Pull request merged**